distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Managed_Resource.enso

```diff
@@ -58,13 +67,19 @@ type Managed_Resource
        ADVANCED
 
        Executes the provided action on the resource managed by the managed
-       resource object.
+       resource object. The action is only invoked if the managed resource
+       has not yet been finalized.
 
        Arguments:
        - action: The action that will be applied to the resource managed by
          resource.
-    with : (Any -> Any) -> Any
-    with self ~action = @Builtin_Method "Managed_Resource.with"
+       - on_missing: Optional action to perform when the value is no longer
+         reachable
+       Returns:
+       Value returned from the `action` or `Nothing` if the managed resource
+       was already finalized
+    with : (Any -> Any) -> Any -> Any
+    with self ~action ~on_missing=Nothing = with_builtin self action on_missing...
```

Comment:
Remind me, what is the semantics of `on_missing...`?

Reply:
The `...` means to send the value without evaluating it - e.g. it should serve as a "lazy action". However the `on_missing` has been removed in 873bac3